### ISSUE TITLE
Error: expected closing ')' running samples170

### ISSUE THREAD

**Issue by zetashift** [Vladar4/libtcod_nim#12]
Using Nim stable and tried it on devel, windows x64.
So running a simple main or the samples I get the following error:

```
λ nim c samples.nim
Hint: used config file 'C:\Users\rskar\.choosenim\toolchains\nim-0.19.0\config\nim.cfg' [Conf]
Hint: system [Processing]
Hint: samples [Processing]
Hint: libtcod [Processing]
Hint: strutils [Processing]
Hint: parseutils [Processing]
Hint: math [Processing]
Hint: bitops [Processing]
Hint: algorithm [Processing]
Hint: unicode [Processing]
..\..\..\..\.nimble\pkgs\libtcod_nim-0.99\src170\parser.nim(117, 5) Error: expected closing ')'
```

Looking at the line there doesn't seem to be anything wrong with it though. 
EDIT: removing the comment made it compile even though nim check threw a bunch of errors at me.

**Comment by Vladar4** [comment]
Thanks for the info! Should be fixed in 690edaab13b1e1991c847961c17a6140efd0918b

**Comment by zetashift** [comment]
Thank you!!
